packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -1517,7 +1517,7 @@ onUpdated(async () => {
 			>
 				<N8nOption
 					v-for="option in parameterOptions"
-					:key="option.value.toString()"
+					:key="`${option.value}`"
```

Comment:
Can we fix the type to prevent this issue happening again?

Reply:
The type of `option.value` (`INodePropertyOptions`), is already `string | number | boolean`. The issue likely comes from Node options definition or `loadOptions`, as @elsmr mentioned.

Reply:
Agree with Mutasem. This probably just hides the issue.

Reply:
I think the type is valid, there is probably a bug in some node(s) where `loadOptions` is not returning a `value`.
We should fix that, but it's hard to track down. Better typing in node code would help here.

For the frontend, I think filtering them out is the cleanest fix 👍

Reply:
Sounds good. Can we filter them out and include a sentry report for that would help us pinpoint the node in question?